docs/demo.md

```diff
@@ -0,0 +1,44 @@
+# Demo
+
+A live demo of the platform is found at: [pori-demo.bcgsc.ca](https://pori-demo.bcgsc.ca). This demo was deployed via docker-compose.
+
+!!! Info "Some Features are Not Available in the Demo Application"
+
+    These include all write operations as well as the automated linking between the kb-matches page in IPR and the GraphKB statements.
+
+!!! Info "Data in the Demo is Limited"
+
+    The data provided in the demo application is a subset of what you might use in your own production application of PORI. The GraphKB data contains partial copies of various open data sets for demonstrative purposes only. This demo should not be used to annotate your own reports. Instead please see the [install section](./install.md) for instructions on setting up and deploying a your own instance of PORI.
+
+## IPR
+
+The demo instance of IPR is at [pori-demo.bcgsc.ca/ipr](https://pori-demo.bcgsc.ca/ipr). This will bring you to the login page of the IPR client application. Use the following credentials to log in to the demo application
+
+- username: `iprdemo`
+- password: `iprdemo`
+
+When you first go to the link above it will redirect you to the keycloak login page. Enter the above credentials
+
+![keycloak login](./images/pori-demo-login.png)
+
+Once you are logged in you will be brought to the main report listing page. This is a table of all the reports your user has access to in IPR. The table can be sorted and filtered.
+
+![report listing](./images/pori-demo-ipr-click-row.png)
+
+By clicking on a row in the above table you will be brought to the corresponding reports main summary page. The different sections of the report can be navigated via the right-hand side bar.
```

Comment:
will this still be true after the client update that adds a button and makes rows selectable instead of clickable?

Reply:
might add 'Once in the report view,' or append 'in the report view' to 'The different sections... right hand side bar.'

Reply:
No but I will update it when we update the version of the client used by the demo